index.bs

```diff
@@ -8644,8 +8664,24 @@ in strict mode to be ignored rather than causing an exception to be thrown.
     <a href="mailto:public-script-coord@w3.org">public-script-coord@w3.org</a>
     mailing list before proceeding.
 
+    <small class="non-normative">
```

Comment:
"advisement" is normative, and thus the specs were the linked APIs are defined end-up in the normative parts of the spec.